.github/workflows/python-publish.yaml

```diff
@@ -0,0 +1,30 @@
+name: python-publish
+on:
+  push:
+    tags:
+      - '^v[0-9]+\.[0-9]+(\.[0-9]+)*'
+    branches:
+      - master
+    paths:
+      - pkg/logging/python_logging/**
```

Comment:
Are you sure that you want to publish on push to master? Or is it just for the testing phase and will be removed later?

Reply:
will be removed later. I will update the comment accordingly.